src/log.h

```diff
@@ -20,6 +20,12 @@
 #include <stdarg.h>
 #include <stdio.h>
 
+#ifdef __GNUC__
+#define ATTR_FORMAT __attribute__((format(printf, 1, 2)))
```

Comment:
I added it because I found 1 case in my code of mismatching arguments, and thought it's just best to add this as well.